spec/polaris-management-service.yml

```diff
@@ -901,6 +903,58 @@ components:
       required:
         - roleArn
 
+    S3StorageConfigInfo:
```

Comment:
Is this specific to s3compat, or is it also meant to be used for s3 itself?

Reply:
I think that roleArn is not mandatory for AWS S3.
Using access and secret key should work with AWS S3 too.
Same naming bad compromise explained previously. I agree to modify in a better suited way and be aligned with Polaris conventions

Reply:
If we can unify these, I think that would be ideal. But I don't know enough about how S3 vs S3Compatible are similar/different to say how possible that is.

Reply:
I think it will not be too hard to unify, in a next step. I miss AWS access to do tests, but for what I know or seen : 

- STS endpoint is a AWS IAM endpoint, in S3 compatible solutions, when it is available, it is merged with S3 endpoint.
- Region is available in S3 compatible solutions, but not used a lot, or mostly implemented to be compliant with aws sdk clients.
  - about the cross region, it is more related to Polaris  and Iceberg with the Iceberg overloaded S3fileIO, but I seen that https://github.com/apache/iceberg/pull/11259 have been merged 3 weeks ago --> so next Iceberg version will be ok soon.
 
MinIO claim that their product API is 100% compatible with AWS S3 API. Almost the same for many alternatives...

- The S3 Compatible implementation could easily propose an optional parameter "arnRole" like the existing aws class, with less regexp patern to allow more flexibility for some implementation where "aws" inside the string is replaced by the product name (exemple "ecs" for DELL ECS)... It could help for a smooth transition